modules/jx/charts.tf

```diff
@@ -35,9 +35,23 @@ resource "helm_release" "jx-git-operator" {
   ]
 }
 
+locals {
+  tf_drift_secret_map = {
+    "ARM_USE_MSI" : "true",
+    "ARM_SUBSCRIPTION_ID" : var.subscription_id,
+    "ARM_TENANT_ID" : var.tenant_id,
+    "ARM_CLIENT_ID" : var.jx_health_client_id,
+    "ARM_CLIENT_SECRET" : var.jx_health_client_secret,
+  }
+}
+
 module "jx-health" {
-  count  = var.enabled ? 1 : 0
-  source = "github.com/jenkins-x/terraform-jx-health?ref=main"
+  count               = var.enabled ? 1 : 0
+  source              = "/Users/chrismellard/go/src/github.com/jenkins-x/terraform-jx-health"
```

Comment:
Should it be a public url instead of a local file?

Reply:
thanks @hervelemeur - tearing my hair out wondering why the local tests worked but didn't remotely ;)